routers/routes/routes.go

```diff
@@ -406,6 +406,13 @@ func RegisterRoutes(m *macaron.Macaron) {
 	reqRepoIssuesOrPullsWriter := context.RequireRepoWriterOr(models.UnitTypeIssues, models.UnitTypePullRequests)
 	reqRepoIssuesOrPullsReader := context.RequireRepoReaderOr(models.UnitTypeIssues, models.UnitTypePullRequests)
 
+	reqRepoWriter := func(ctx *context.Context) {
```

Comment:
Could you change that to something like `RepoIssueWriter`? IMHO `RepoWriter` indicates the user can also push to the repo etc.
The name is somewhat misleading.